scheduler/scheduler-server/src/test/java/performancetests/recovery/JobRecoveryTest.java

```diff
@@ -65,9 +67,11 @@
 
     private static URL runningJob = JobRecoveryTest.class.getResource("/functionaltests/descriptors/Job_running.xml");
 
+    private static final Logger LOGGER = Logger.getLogger(JobRecoveryTest.class);
+
     @Parameters
     public static Collection<Object[]> data() {
-        return Arrays.asList(new Object[][] { { 5, 1000 }, { 100, 5000 } });
+        return Arrays.asList(new Object[][] { { 10, 1000 }, { 100, 2000 }, { 500, 5000 }, { 1000, 10000 } });
```

Comment:
This data object Collection<Object[]>, can you explain e.g. in the name or as a comment what it represents. That will improve the test readability a lot.